### DISCUSSION TITLE
Student Card Image Problem

### DISCUSSION THREAD

**Discussion by Muhammad-Zain-PLAYER** [community/community#146897] [GitHub Education]
### Hi everyone,

My webcam image quality is not good. When I use my smartphone, it does not access my back camera. It only accesses the front camera, which is not of good quality. What should I do?

**Comment by dravenlocke87** [comment]
Hey, I think you should check the camera settings in the app you're using. because most apps have an option to switch between front and back cameras. plus make sure the app has permission to access both cameras in your phone’s settings. If that doesn’t work, try restarting the app or even reinstalling it.

**Comment by Mohrip** [comment]
> ### Hi everyone,
> My webcam image quality is not good. When I use my smartphone, it does not access my back camera. It only accesses the front camera, which is not of good quality. What should I do?

To fix the camera issue, check the app or browser settings to ensure the back camera is selected and all permissions are granted. If that doesn't work, try using a third-party app like DroidCam or EpocCam to access your back camera, or update/reinstall the app. If the problem persists, report it on GitHub with details about your device, OS, and browser.

**Reply by vadimsabadosh** [reply]
I'm using safari, what tan app you're describing here? in github app on iOS I cannot find education things like on the website.
